aperturedb/EntityDataCSV.py

```diff
@@ -70,3 +72,43 @@ def getitem(self, idx):
     def validate(self):
         if self.header[0] != ENTITY_CLASS:
             raise Exception("Error with CSV file field: " + ENTITY_CLASS)
+
+# Used when a csv has a single entity type that needs to be deleted
+
+
+class SingleEntityDeleteCSV(CSVParser.CSVParser):
+    def __init__(self, entity_class, filename, df=None, use_dask=False):
+        super().__init__(filename, df=df, use_dask=use_dask)
+        self.command = "Delete" + entity_class
```

Comment:
Can it be done like this  ? I thought the command would DeleteEntity : { "with_class" : entity }

Reply:
I think Luis had mentioned earlier that DeleteEntity won't work properly for Image or Video since they do the delete in their classes and Entity won't run that.